packages/sign-client/src/controllers/engine.ts

```diff
@@ -916,6 +1351,25 @@ export class Engine extends IEngine {
     throw new Error(message);
   };
 
+  private shouldIgnorePairingRequest = (params: { topic: string; requestMethod: string }) => {
```

Comment:
Can we add this fn signature type to `EnginePrivate` pls for consistency and define `boolean` as return type explicitly so it can't return `void` accidentally if someone later touches the early or default returns here 🙏